src/main/java/uk/gov/pay/api/resources/PaymentsResource.java

```diff
@@ -195,32 +193,45 @@ public PaymentEventsResponse getPaymentEvents(@Parameter(hidden = true) @Auth Ac
     )
     public Response searchPayments(@Parameter(hidden = true)
                                    @Auth Account account,
-                                   @Parameter(description = "Your payment reference to search (exact match, case insensitive)")
+                                   @Parameter(description = "Returns payments with `reference` values exactly matching your specified value.")
                                    @QueryParam("reference") String reference,
-                                   @Parameter(description = "The user email used in the payment to be searched")
+                                   @Parameter(description = "Returns payments with matching `email` values. You can send full or partial email addresses. " +
+                                           "`email` is the paying user’s email address.")
                                    @QueryParam("email") String email,
-                                   @Parameter(description = "State of payments to be searched. Example=success", example = "success",
+                                   @Parameter(description = "Returns payments in a matching `state`. `state` reflects where a payment is in the " +
+                                           "[payment status lifecycle](https://docs.payments.service.gov.uk/api_reference/#payment-status-lifecycle).", example = "success",
                                            schema = @Schema(allowableValues = {"created", "started", "submitted", "success", "failed", "cancelled", "error"}))
                                    @QueryParam("state") String state,
-                                   @Parameter(description = "Card brand used for payment. Example=master-card")
+                                   @Parameter(description = "Returns payments paid with a particular card brand.")
                                    @QueryParam("card_brand") String cardBrand,
-                                   @Parameter(description = "From date of payments to be searched (this date is inclusive). Example=2015-08-13T12:35:00Z")
```

Comment:
Can we add `example="2015-08-13T12:35:00Z" ` attribute as the example is no longer in the description